### PR TITLE
Update nGraph PythonAPI - P1 ops

### PR DESCRIPTION
**Pull request by Ewa21**
In this PR following ops are updated or added to Python API:
 * Broadcast,
* Clamp,
* Concat,
* LRN,
* NonMaxSuppression,
* NonZero,
* ReLU,
* ROIAlign,
* ShapeOf
* TopK

### CODE REVIEW

File: python/src/ngraph/ops.py [hunk 1]
```diff
@@ -2279,14 +2375,14 @@ def sigmoid(data):  # type: (Node) -> Node
...
     """Return a node which produces a tensor containing the shape of its input data.
 
-    :param data: The tensor containing the input data
+    :param data: The tensor containing the input data.
+    :para output_type: Output element type.
     :return: ShapeOf node
     """
-    # TODO: update to use opset3 by default
-    return _get_node_factory('opset1').create('ShapeOf', [data])
+    return _get_node_factory('opset3').create('ShapeOf', [data], {'output_type': output_type})
```
Comment (postrational) [inline python/src/ngraph/ops.py:2385]: We're already switching to `opset3` by default, so you can remove the explicit call here.

File: python/test/ngraph/test_ops_reshape.py [hunk 2]
```diff
@@ -97,17 +97,31 @@ def test_constant_from_integer_array(val_type, range_start, range_end):
...
-    expected = np.add(a_arr, b_arr)
+    data_parameter = ng.parameter(data_shape, name='Data', dtype=np.float32)
+    target_shape_parameter = ng.parameter(target_shape_shape, name='Target_shape', dtype=np.int64)
 
-    assert np.allclose(result, expected)
+    node = ng.broadcast(data_parameter, target_shape_parameter)
+
+    assert node.get_type_name() == 'Broadcast'
+    assert node.get_output_size() == 1
+
```
Comment (mitruska) [inline python/test/ngraph/test_ops_reshape.py:111]: Can we check expected output shape here?

Comment (Ewa21) [reply python/test/ngraph/test_ops_reshape.py:111]: Unfortunately, we can't. Broadcast's output is a tensor with a dynamic shape. Currently, we don't have  any `get_output_partial_shape` function in Python API.

File: python/src/ngraph/ops.py [hunk 3]
```diff
@@ -1240,23 +1240,25 @@ def squared_difference(x1, x2, auto_broadcast='NUMPY', name=None):
...
     :return: New node with broadcast shape.
     """
-    # TODO: update to use opset3 by default
-    return _get_node_factory('opset1').create('Broadcast',
-                                              [node, new_shape, broadcast_axes],
-                                              {'auto_broadcast': auto_broadcast})
+    inputs = [data, as_node(target_shape)]
+    if broadcast_spec.upper() == 'EXPLICIT':
+        inputs.append(as_node(axes_mapping))
+    return _get_node_factory('opset3').create('Broadcast',
```
[the remark this answers is not in the document]
Comment (Ewa21) [reply python/src/ngraph/ops.py]: Done

File: python/src/ngraph/ops.py [hunk 4]
```diff
@@ -2141,11 +2146,68 @@ def argmin(data,    # type: Node
...
+    return _get_node_factory().create('NonMaxSuppression', inputs, attributes)
+
+
+@nameable_op
+def non_zero(data,                # type: Node
+             output_type='i64',   # type: str
+             ):
+    # type: (...) -> Node
+    """Return a node which performs NonZero.
+
```
[the remark this answers is not in the document]
Comment (Ewa21) [reply python/src/ngraph/ops.py:2238]: Done

File: python/src/ngraph/ops.py (hunk 1 above)
Comment (Ewa21) [reply python/src/ngraph/ops.py]: Done

**Pull request merged**